contentcuration/contentcuration/assets/shared/styles/main.less

```diff
@@ -0,0 +1,4 @@
+body {
+  font-family: 'Noto Sans', Verdana, sans-serif;
```

Comment:
I am starting minimally in terms of porting styles across. This was important for overall look and feel of the page, but I've not found anything else that is necessary.